build/charts/antrea/values.yaml

```diff
@@ -74,24 +74,13 @@ ovs:
   hwOffload: false
 
 secondaryNetwork:
-  ovs:
-    # -- Enable OVS bridge configuration for secondary network.
-    enable: false
-    # -- Secondary network OVS integration bridge name.
-    integrationBridgeName: "br-secnet-int"
-    # -- Secondary network OVS transport bridge name.
-    transportBridgeName: "br-secnet-trans"
-    # -- Datapath type to use for the OpenVSwitch bridge created by Antrea. Supported values are:
-    # - system
-    # - netdev
-    # -- 'system' is the default value and corresponds to the kernel datapath. Use 'netdev' to run
-    # OVS in userspace mode. Userspace mode requires the tun device driver to be available.
-    datapathType: "system"
-    # -- Name of the OVS patch port which connects the integration and transport bridge.
-    patchPort: "br-secnet-patch0"
-  # -- Tunnel protocol used for encapsulating traffic across Nodes. It must be one
-  # of "geneve", "vxlan", "gre", "stt".
-  tunnelType: "geneve"
```

Comment:
let's have a separate PR to remove this. It will be easier to discuss whether we need to worry about backwards-compatibility.

Reply:
The thing is that my new definition duplicates many fields with the current one. It looks pretty strange to keep both. Could you check: https://github.com/antrea-io/antrea/pull/5279/files#diff-1151756c4182675d1902affa33845d6ad94811a0f59023c59491c1a35959a735L371.

Reply:
I was suggesting a different PR that we would merge quickly, before this one. You can even generate it by reverting https://github.com/antrea-io/antrea/pull/4440, which makes things easy to track IMO.

There is definitely no issue with removing the Helm values. Usually, a non backwards-compatible change to the Go config type is more of a problem, but in this case we never added these configuration parameters to antrea.yml, so it should be fine.

Reply:
To confirm, you suggest to create a separate PR to revert #4440, which can be merged first before this one?

Reply:
Yes

Reply:
Create #5341 to revert #4440.